src/js/nav/sourceOfTruth.js

```diff
@@ -19,6 +19,11 @@ module.exports = (cachePrefix) => {
 
     instance.interceptors.response.use((response) => response.data || response);
 
-    return instance.get('https://raw.githubusercontent.com/'
-    + 'RedHatInsights/cloud-services-config/master/main.yml');
+    // TODO: Add prefix (/beta) depending on environment
+    let prefix = '';
+    if (window.location.pathname.indexOf('/beta')) {
+        prefix = '/beta';
+    }
+
+    return instance.get(window.location.host + prefix + '/config/main.yml');
```

Comment:
Use `window.location.origin` to include protol as well.

Reply:
Oops - just pushed a commit that added the protocol, but your solution is cleaner!